icu4j/maven-migration/toMaven.sh

```diff
@@ -36,6 +47,13 @@ function rmFileIfExists() {
   [ -f "$1" ] && rm "$1"
 }
 
+function copyRecursive() {
+  SRC=$1
+  DEST=$2
+
+  cp -R "$1" "$2"
+}
+
 function safeMoveDir() {
   export FOLDER_NAME=$1
```

Comment:
s/export/local/
(I know that's what it was like, but meantime I learned better :-)
https://tldp.org/LDP/abs/html/localvar.html

Reply:
Done.